sections/02_intro.md

```diff
@@ -33,146 +33,128 @@ less fruitful to apply.
-cancer patients [@doi:10.1142/9789814644730_0014]. Ciresan et al. utilized
-convolutional neural networks (CNN) to count mitotic divisions in
-histological images; a feature that is highly correlated with disease
-outcome [@doi:10.1007/978-3-642-40763-5_51]. Despite these recent advances, a
-number of challenges exist in this area of research, such as the integration
-of disparate types of data, including electronic health records (EHR),
-imaging and histology data, and molecular omics data.
+breast cancer patients: denoising autoencoders (DA), an unsupervised approach,
+can be used to cluster breast cancer patients [@doi:10.1142/9789814644730_0014],
+and convolutional neural networks (CNN) can help count mitotic divisions, a
+feature that is highly correlated with disease outcome, in histological images
+[@doi:10.1007/978-3-642-40763-5_51]. Despite these recent advances, a number of
+challenges exist in this area of research, most notably the integration of
+molecular and imaging data with other disparate types of data such as electronic
+health records (EHR).
 
 #### Fundamental Biological Study
 
-Deep learning can be applied to answer more fundamental
-biological questions, and is especially suited to leveraging
-large amounts of data from high throughput omics studies. One
-classic biological problem where machine learning has been extensively
-applied is the prediction of molecular targets. Recent advances using deep
-learning have shown higher accuracy in determining molecular targets. For
-example, Lee et al. used deep recurrent neural networks (RNN) to predict gene
-targets of micro-RNAs [@doi:10.1109/icnn.1994.374637]. Wang et al. used a
-residual CNN to predict protein-protein contact on a genome-wide scale
-[@doi:10.1101/073239]. Other biological questions that have been investigated
-include the prediction of protein secondary structure based on sequence data
-[@doi:10.1109/tcbb.2014.2343960 @doi:10.1038/srep18962],
-recognition of functional genomic elements such as enhancers and
-promoters [@doi:10.1101/036129 @doi:10.1007/978-3-319-16706-0_20
-@doi:10.1093/nar/gku1058], predicting the deleterious effects of nucleotide
-polymorphisms [@doi:10.1093/bioinformatics/btu703], etc.
+Deep learning can be applied to answer more fundamental biological questions; it
+is especially suited to leveraging large amounts of data from high-throughput
+"omics" studies. One classic biological problem where machine learning, and now
+deep learning, has been extensively applied is molecular target prediction. For
+example, deep recurrent neural networks (RNN) have been used to predict gene
+targets of microRNAs [@doi:10.1109/icnn.1994.374637], and residual CNNs have been
+applied to predict protein-protein contacts on a genome-wide scale
+[@doi:10.1101/073239]. Some of the many recent exciting applications of deep
+learning include the prediction of protein secondary structure based on sequence
```

Comment:
I suggest we remove the phrase `the prediction of protein secondary structure based on sequence data [@doi:10.1109/tcbb.2014.2343960 @doi:10.1038/srep18962],` because that's what the contact map problem is in the line above.